ts/parts/Interaction.ts

```diff
@@ -884,58 +884,58 @@ extend(Chart.prototype, /** @lends Chart.prototype */ {
 
                 axis.panningState = panningState;
 
-                // It is not necessary to calculate extremes on ordinal axis,
-                // because the are already calculated, so we don't want to
-                // override them.
-                if (!axisOpt.ordinal) {
```

Comment:
Instead of changing logic, would be it sufficient to change to `!axis.isOrdinal`? That flag determines if ordinal positions were calculated, or if data on the chart is equally spaced and `ordinal` option does nothing.